README.md

```diff
@@ -67,7 +67,7 @@ Once you have Carthage [installed](#installing-carthage), you can begin adding f
   $(SRCROOT)/Carthage/Build/iOS/ReactiveCocoa.framework
   ```
 
-  This script works around an [App Store submission bug](http://www.openradar.me/radar?id=6409498411401216) triggered by universal binaries.
+  This script works around an [App Store submission bug](http://www.openradar.me/radar?id=6409498411401216) triggered by universal binaries as well as ensures that necessary bitcode-related files are also copied.
```

Comment:
Actually, that's a bit repetitive on the word "also". So maybe it doesn't sound as good.